src/MongoDB.Driver.Legacy/MongoServerSettings.cs

```diff
@@ -696,8 +724,14 @@ public static MongoServerSettings FromClientSettings(MongoClientSettings clientS
             serverSettings.ServerSelectionTimeout = clientSettings.ServerSelectionTimeout;
             serverSettings.SocketTimeout = clientSettings.SocketTimeout;
             serverSettings.SslSettings = (clientSettings.SslSettings == null) ? null : clientSettings.SslSettings.Clone();
+            serverSettings.TlsInsecure = clientSettings.TlsInsecure;
+#pragma warning disable 618
             serverSettings.UseSsl = clientSettings.UseSsl;
+#pragma warning restore 618
+            serverSettings.UseTls = clientSettings.UseTls;
+#pragma warning disable 618
             serverSettings.VerifySslCertificate = clientSettings.VerifySslCertificate;
+#pragma warning restore 618
```

Comment:
I don't think we need to be copying `UseSsl` and `VerifySslCertificate` because they are just aliases.

Reply:
Done